src/lib/mongoBackend/mongoQueryTypes.cpp

```diff
@@ -149,6 +149,126 @@ static long long countEntities(const std::string& tenant, const std::vector<std:
 
 /* ****************************************************************************
 *
+* mongoEntityTypesValues -
+*
+* "Simplified" version of mongoEntityTypes(), using a simpler aggregation command
+* and the processing logic afterwards. Note hat apiVersion is not included in this
+* operation as it can be only used in NGSIv2
+*/
+HttpStatusCode mongoEntityTypesValues
+(
+  EntityTypeVectorResponse*            responseP,
+  const std::string&                   tenant,
+  const std::vector<std::string>&      servicePathV,
+  std::map<std::string, std::string>&  uriParams,
+  unsigned int*                        totalTypesP
+)
+{
+  unsigned int   offset         = atoi(uriParams[URI_PARAM_PAGINATION_OFFSET].c_str());
+  unsigned int   limit          = atoi(uriParams[URI_PARAM_PAGINATION_LIMIT].c_str());
```

Comment:
Do we have any check anywhere to verify that offset and limit are actually integers?